### ISSUE TITLE
Some Awards Not Listed for Profile Restoration

### ISSUE THREAD

**Issue by MrPliffles** [emperorshammer/issues-incoming-queue#230]
https://tc.emperorshammer.org/management/medals.php

Missing options like the DFC and IAR. Probably because my profile is an EH profile and those are TC awards. New variation of the problem we've run into a few times now, I'm guessing.

![⁄ 2024-09-07 01 10 04](https://github.com/user-attachments/assets/82fa6a61-39ac-4b0e-98c3-669e22f69523)

**Comment by MrPliffles** [comment]
But then again, the Iron Star, ISM, and PC are missing as well. Hmm.

**Comment by turtlexiv** [comment]
Searching for them doesn't yield anything I take it?

**Comment by MrPliffles** [comment]
Nope. I get Imperial Cross all the way up to "imperial s" and then I get "No matches found".

**Comment by turtlexiv** [comment]
Weird. Same problem for me.
